test/graphql.expenses.test.js

```diff
@@ -328,6 +335,157 @@ describe('GraphQL Expenses API', () => {
       expect(res.errors).to.not.exist;
     }); /* End of "creates a new expense logged in and send email to collective admin for approval" */
 
+    it('creates a new expense greater than 600 USD but DO NOT create Comment because host is not USD based', async () => {
```

Comment:
I'd create a new test file `bot.irs.test.js` to put this -- that will also make it easier for people to create other bots.

Reply:
I agree with you somehow but at the same time it's also testing the Graphql expenses creation(and it's additional feature) so I'm kinda not sure if moving this to a new file would make it somehow confusing. Do you disagree?